extensions/amp-analytics/amp-analytics.md

```diff
@@ -291,6 +291,39 @@ In this example, all requests are valid.
 
 Some analytics providers have an already-provided configuration, which you use via the `type` attribute. If you are using an analytics provider, you may not need to include requests information. See your vendor documentation to find out if requests need to be configured, and how.
 
+##### Request Origin
+The top-level `requestOrigin` property accepts an absolute URL and defines the origin for requests. If `requestOrigin` is declared, the origin will be extracted from the value and it will be prepended to every outgoing request. `requestOrigin` will not be encoded and also accepts variables.
```

Comment:
Might worth clarifying that `requestOrigin` and request url will be added together. So that user is responsible for making sure that `requestOrigin` + `baseUrl` forms the valid and correct outgoing url.